airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py

```diff
@@ -243,6 +243,15 @@ def backoff_time(self, response: requests.Response) -> Optional[float]:
         """
         return None
 
+    def error_message(self, response: requests.Response) -> str:
```

Comment:
I don't love the idea of a new method to the CDK to support a low-code use case, but it could still be useful for other connectors that need to override retryable errors

Reply:
Interfacing with HttpStream is always tricky. 

I'm in favor of those refactors so long as the default behavior does not change so we don't impact existing connectors.